packages/sdk/src/dataProtector/processProtectedData.ts

```diff
@@ -24,14 +25,15 @@ export const processProtectedData = async ({
   args,
   inputFiles,
   secrets,
+  workerpool,
```

Comment:
added `workerpool` option to override the default value as it was not possible (this is required for local testing as we don't control the default pool)